.github/workflows/ci.yaml

```diff
@@ -270,4 +270,98 @@ jobs:
         curl -s https://codecov.io/bash | bash
         echo "Uploaded"
 
+  linux-fpga:
```

Comment:
This is a new job created as part of the MIGraphx runtime, basically a copy of the existing linux job seen above with some modification.

Good resource for this is: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions